checks/raw/fuzzing.go

```diff
@@ -208,6 +209,10 @@ func Fuzzing(c *checker.CheckRequest) (checker.FuzzingData, error) {
 
 	langs, err := c.RepoClient.ListProgrammingLanguages()
 	if err != nil {
+		// ListProgrammingLanguages is unsupported by local client so just return
+		if errors.Is(err, clients.ErrUnsupportedFeature) {
+			return checker.FuzzingData{Fuzzers: detectedFuzzers}, nil
+		}
```

Comment:
we miss out on most of the file-based detection if we return early here. (only clusterfuzzlite is detected).

I wonder if it makes sense to scan for every language if the client doesn't have the info